.bazelrc

```diff
@@ -0,0 +1,6 @@
+# `.bazelrc` is a Bazel configuration file.
+# https://docs.bazel.build/versions/master/best-practices.html#using-the-bazelrc-file
+
+# Enable rustfmt for all targets in the workspace
+build --aspects=//rust:defs.bzl%rustfmt_aspect
```

Comment:
This now applies to all bazel builds, I'd prefer to have it applied only when `--config=rustfmt` is passed by writing:

```
build:rustfmt --aspects=//rust:defs.bzl%rustfmt_aspect
build:rustfmt --output_groups=+rustfmt_checks
```

Also below. I thought that's what we agreed on in the discussion before?

Reply:
It is, I'd forgotten about it... Had a busy, distracting week 😅 

Added the config

Reply:
No problem, thanks for working on this!